soc/nxp/rw/flexspi_clock_setup.c

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
This file might is not needed for initial support for this SoC and might be for some future work. Please delete from this PR and add when support for appropriate component is added.

Reply:
the file is needed to clock the flexspi most appropriately for XIP which is the way to execute code on the SOC